content/millennium/r4/base/management/encounter.md

```diff
@@ -98,23 +98,24 @@ _Implementation Notes_
 
 ### Parameters
 
-Name          | Required?                                                     | Type          | Description
+Name          | Required                                                     | Type          | Description
 --------------|--------------------------------------------------------------------------------|---------------|---------------------------------------------------------------------------------------------------
- `_id`        | This or `patient` or `subject` or `account` or `identifier` or `pageContext`   | [`token`]     | The logical resource id associated with the Encounter. Example: `7891`
+ `_id`        | This or `patient` or `subject` or `account` or `identifier` or `pageContext`   | [`token`]     | The logical resource ID associated with the encounter. Example: `7891`
```

Comment:
@summerlcollins, should this be updated to match the "Conditionally" formatting?

Reply:
I've added this change to both R4 and DSTU2 versions and updated the screenshots.

Tagging @cldeitrich since Summer is OOO

Reply:
Yes, we should apply the same formatting for consistency.